content/blog/2024/09/25/2024-09-25-board-officer-election-announcement.adoc

```diff
@@ -0,0 +1,132 @@
+
+== Nominees
+
+We will be electing three members of the Jenkins Governance Board from among six candidates.
+The six candidates for the Jenkins Governance Board are:
+
+* Alex Earl - link:https://contributors.jenkins.io/pages/contributors/alex-earl/[contributor spotlight] and link:/blog/authors/slide_o_mix/[author biography]
+* Alexander Brandes - link:https://contributors.jenkins.io/pages/contributors/alexander-brandes/[contributor spotlight] and link:/blog/authors/notmyfault/[author biography]
+* Kris Stern - link:https://contributors.jenkins.io/pages/contributors/kris-stern/[contributor spotlight] and link:/blog/authors/krisstern/[author biography]
+* Oleg Nenashev - link:/blog/authors/oleg_nenashev/[author biography]
+* Stefan Spieker - link:https://contributors.jenkins.io/pages/contributors/stefan-spieker/[contributor spotlight]
+* Valentin Delaye - link:https://contributors.jenkins.io/pages/contributors/valentin-delaye/[contributor spotlight]
+
+We will be electing the Jenkins Release Officer from among two candidates.
+The two candidates for Jenkins Release Officer are:
+
+* Alex Earl - link:https://contributors.jenkins.io/pages/contributors/alex-earl/[contributor spotlight] and link:/blog/authors/slide_o_mix/[author biography]
+* Tim Jacomb - link:https://contributors.jenkins.io/pages/contributors/tim-jacomb/[contributor spotlight] and link:/blog/authors/timja/[author biography]
+
+Candidate statements, affiliations, and profile links will be provided later.
+
+The following officer positions are uncontested, so no election is needed for:
+
+* Damien Duportal - Jenkins Infrastructure Officer - link:/blog/authors/dduportal/[author biography]
+* Wadeck Follonier - Jenkins Security Officer - link:/blog/authors/wadeck/[author biography]
+* Alyssa Tong - Jenkins Events Officer - link:https://contributors.jenkins.io/pages/contributors/alyssa-tong/[contributor spotlight] and link:/blog/authors/alyssat/[author biography]
+* Kevin Martens - Jenkins Documentation Officer - link:https://contributors.jenkins.io/pages/contributors/kevin-martens/[contributor spotlight] and link:/blog/authors/kmartens27/[author biography]
+
+=== Contributing
+
+As shown on the link:/participate/[Participate and Contribute] page, there are many different ways to contribute to Jenkins. You can contribute by:
+
+* link:/participate/connect/[Connecting with the community]
+* link:/participate/meet/[Joining or organizing a meetup]
+* link:/participate/code/[Contributing code to Jenkins]
+* link:/participate/help/[Helping Jenkins users]
+* link:/doc/developer/internationalization/[Translating Jenkins resources]
+* link:/participate/test/[Testing Jenkins core and plugins]
+* link:/participate/document/[Contributing to Jenkins documentation]
+* link:/participate/design/[Creating art or updating the Jenkins UI]
+* link:/participate/review-changes/[Reviewing open pull requests]
+* link:/donate/[Donating to Jenkins]
```

Comment:
Optional suggestion: remove `You can contribute by:` and change all the verbs to be in active form (ex: "Connect with the community")

Reply:
Thanks!  Done in 8a5228c1599f08efa2e2a6b5884087fd6e28b066